uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/UserRoleItem.php

```diff
@@ -31,8 +31,8 @@ public function render() {
     $roles = implode(', ', $this->cmsUserRole->getRoles());
 
     $userRolesMarkup = '
-      <p><span class="crm_contact_action_menu__bold_text">Roles: </span> 
-        <span class="crm_contact_action_menu__grey_text">%s</span>
+      <p><span class="crm_contact-actions--bold">Roles: </span> 
+        <span class="crm_contact-actions--dark-grey">%s</span>
```

Comment:
what's this class for? Please add screenshot

Reply:
<img width="355" alt="screen shot 2018-01-24 at 2 01 32 pm" src="https://user-images.githubusercontent.com/6307362/35321582-4d54bb4e-010f-11e8-8874-e6be37978b20.png">

It makes the test dark grey. The color was suggested by @jon. The class is used in the tests following "User"  and  "Roles"  above.

Reply:
why do we need entire new classes for such small things? Can't you have the parent set this properties on the children?

Reply:
Removed them as they no longer needed now after the use of  (`<dl>`).